Readability.js

```diff
@@ -732,7 +732,12 @@ Readability.prototype = {
             candidates.push(ancestor);
           }
 
-          ancestor.readability.contentScore += contentScore / (level === 0 ? 1 : level * 2);
+          // Node score divider:
+          // - parent:             1 (no division)
+          // - grandparent:        2
+          // - great grandparent+: 6
+          var scoreDivider = level === 0 ? 1 : level === 1 ? 2 : 6;
```

Comment:
I'm vaguely uncomfortable with the magic scoring here, but sure, we can do this...